pkg/ecmh/ecmh.go

```diff
@@ -0,0 +1,60 @@
+package ecmh
+
+import "github.com/bwesterb/go-ristretto"
+
+// MultisetHash is a multiset hash based on ECMH
+// implementated using ristretto points.
+type MultisetHash struct {
```

Comment:
implements the multihash as it is implemented here: https://github.com/tablelandnetwork/hhash/blob/main/src/lib.rs#L28